samples/bluetooth/hci_spi/README.rst

```diff
@@ -38,3 +38,107 @@ Zephyr's Bluetooth HCI driver core; see the help associated with the
 Refer to :ref:`bluetooth-samples` for general Bluetooth information, and
 to :ref:`96b_carbon_nrf51_bluetooth` for instructions specific to the
 96Boards Carbon board.
+
+Using the controller with the Zephyr host
+=========================================
+
+This describes how to hook up a board running this sample to a board running
+an application that uses the Zephyr host.
+Here the example is with two nRF52840dk boards.
+All wires are connected to the same port/pin on the other side, save for the reset line, which is connected to a GPIO on the host and the SoC reset line on the controller.
```

Comment:
Could you reformat this a bit, since it seems highly inconsistent with the line lengths?

Reply:
It's to help with the reading. I have trouble with thick walls of text. I'll add a newline and reflow if that's ok